charts/base-cluster/templates/monitoring/kube-prometheus-stack/_grafana-config.yaml

```diff
@@ -13,6 +13,11 @@ securityContext:
     type: RuntimeDefault
 containerSecurityContext: {{- include "base-cluster.prometheus-stack.containerSecurityContext" (dict) | nindent 2 }}
 resources: {{- include "common.resources" .Values.monitoring.grafana | nindent 2 }}
+{{- if .Values.monitoring.grafana.persistence.enabled }}
+persistence:
```

Comment:
Missing storageClass, see https://github.com/SyeKlu/teutonet-helm-charts/blob/main/charts/base-cluster/templates/nfs-server-provisioner/nfs-server-provisioner.yaml#L22-L25

Reply:
Oh I was somehow blind :grin: I looked at the grafana helm [values](https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml#L367-L383) and didn't notice the comment with the storageclassname :facepalm: .